core/controllers/galleries.py

```diff
@@ -113,6 +113,10 @@ def get(self):
         explorations_list = (
             summary_services.get_displayable_exp_summary_dicts_matching_ids(
                 exp_ids))
+        # TODO(mgowano): Remove '' in query to allow for collection searches
+        collections_list = (
+            collection_services.get_displayable_collections_matching_query(
```

Comment:
I have modified this and made it consistent with exploration summary retrieval but still the problem persists. The issue seem to be with language_code. When I put the `query_string`, two calls to `get_displayable_col_summary_dicts_matching_ids` happen. 

The call has `query_string` as an empty string, with this a list with one collection is returned.

The second call has `query_string` as language_code=("en"), with this an empty list is returned. Could there be issues with the way the language_code has been implemented in collections?